db/migration/1712842552502-AddPostsSlugIndex.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddPostsSlugIndex1712842552502 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+            ALTER TABLE posts ADD INDEX idx_posts_slug (slug(100));
```

Comment:
I think we should bump this to 130 maybe? The longest slug in my snapshot is 122 chars long. As far as I understand indices in mysql this would still work but why not make it a bit bigger and then all values will fit.

Reply:
Hm, I honestly don't think it's necessary (unless we would want this index to be unique).
Otherwise, there are quite certainly not gonna be two slugs whose prefixes of length 100 are the same length, and then it's just the same.